packages/nimble-components/src/number-field/styles.ts

```diff
@@ -29,7 +28,6 @@ export const styles = css`
         outline: none;
         user-select: none;
         color: ${bodyFontColor};
-        height: calc(${labelHeight} + ${controlHeight});
```

Comment:
Would we not want to make this change more broadly? Why would this only apply to the text-field?

Reply:
+1 the PR linked from the issue describes how the textarea has the same fix. Seems like it should be confirmed that we are not playing whack-a-mole

Reply:
The problem only applies to text fields and number fields. The only other control that has an optional label that is vertically stacked on the control is the text area. The text area already doesn't set an explicit height, so its height is calculated correctly whether the label is there or not.